db/data/db.json

```diff
@@ -3,28 +3,88 @@
+                "banner": "the_matrix",
+                "price": 4.0
+            },
+            {
+                "name": "Forrest Gump",
+                "description": "A slow-witted but kind-hearted man named Forrest Gump becomes an unwitting participant in some of the biggest events in American history, while also pursuing his childhood sweetheart, Jenny.",
+                "banner": "forrest_gump",
+                "price": 3.0
+            },
+            {
+                "name": "Jurassic Park",
+                "description": "A wealthy entrepreneur creates a theme park on an island where genetically cloned dinosaurs roam free. When a group of scientists and visitors visit the park, they quickly discover that the dinosaurs are not as safe as they seem.",
+                "banner": "jurassic_park",
+                "price": 5.0
+            },
+            {
+                "name": "The Silence of the Lambs",
+                "description": "An FBI trainee named Clarice Starling is tasked with interviewing a notorious serial killer named Hannibal Lecter in order to gain insight into another killer known as 'Buffalo Bill'.",
+                "banner": "the_silence_of_the_lambs",
+                "price": 4.0
+            },
+            {
+                "name": "The Lion King",
+                "description": "A young lion prince named Simba must overcome the loss of his father and the betrayal of his uncle to take his rightful place as king of the Pride Lands.",
+                "banner": "the_lion_king",
+                "price": 4.0
+              },
+              {
+                "name": "The Fellowship of the Ring",
+                "description": "A hobbit named Frodo Baggins is entrusted with a powerful ring that could ultimately destroy the world. He and a fellowship of other creatures must journey across Middle-earth to destroy the ring and prevent it from falling into the wrong hands.",
+                "banner": "the_fellowship_of_the_ring",
+                "price": 5.0
+              },
+              {
+                "name": "Avengers: Endgame",
+                "description": "The surviving members of the Avengers and their allies work to reverse the damage caused by Thanos in Infinity War and restore balance to the universe.",
+                "banner": "avengers_endgame",
+                "price": 6.0
+              },
+              {
+                "name": "The Terminator",
+                "description": "In the future, an artificial intelligence called Skynet sends a cyborg assassin back in time to kill Sarah Connor, the woman who will give birth to the human resistance. A soldier named Kyle Reese is also sent back to protect her.",
+                "banner": "the_terminator",
+                "price": 4.0
+              }
         ],
         "room": [
             {
```

Comment:
Room doesn't have the expected data fields (`id`, `rows`, `cols`)

Reply:
Thanks for the review, I have fixed the problem by removing the "name" field as it was not needed in this commit [a453516](https://github.com/jkutkutOrg/Android-CineHub/pull/36/commits/a453516afc35fa5629ee220c09dab91987933510).

As for the other fields, I assume that @Jkutkut creates the IDs incrementally in the database (reference [db model](https://github.com/jkutkutOrg/Android-CineHub/blob/wip-database_initial_load/docs/database_model.md)), in case this is not the case, I will add the ids manually, please let me know.

Reply:
> As for the other fields, I assume that @Jkutkut creates the IDs incrementally in the database (reference [db model](https://github.com/jkutkutOrg/Android-CineHub/blob/wip-database_initial_load/docs/database_model.md)), in case this is not the case, I will add the ids manually, please let me know.

Yes, IDs are expected to be provided by the DB. I tried to remark that `name` was unnecessary as it could be dynamically generated in the client app.